examples/config.example.py

```diff
@@ -25,65 +25,50 @@
 # Whether the bot is for testing, if true, stats and errors will not be posted
 testing = True
 
-# PUBSUB channel for Redis
-ipc_channel = ""
-
-# Postgres database credentials
-database = {
-    "database": "",
-    "user": "",
-    "password": "",
-    "host": "",
-    "port": 5432,
-}
+# AMQP credentials
+amqp_url = ""
+
+# Redis credentials
+redis_url = ""
```

Comment:
Not possible as `aioredis` does not support this. 
![image](https://user-images.githubusercontent.com/56286157/111022050-4dd14880-840b-11eb-8382-adf240075346.png)

Reply:
You can parse the address yourself...

Reply:
What's the need to use the dictionary when it does not even fit the context?

Reply:
Besides, what's wrong with using a connection string instead?

Reply:
A dictionary fits the context of this config file. Since you asked:
1. What the library asks for may not be the best, for example, in this case.
2. For consistency with the previous configurations so users self-hosting do not need to make changes.
3. It is much easier to specify a single config, when used with twilight-dispatch.

Please do not ask further unnecesary questions.